firefox-ios/firefox-ios-tests/Tests/XCUITests/HistoryTests.swift

```diff
@@ -439,7 +439,7 @@ class HistoryTests: BaseTestCase {
             mozWaitForElementToExist(app.tables.cells.staticTexts[entry])
         }
         mozWaitForElementToNotExist(app.staticTexts["Today"])
-        mozWaitForElementToExist(app.staticTexts["Older"])
+        mozWaitForElementToExist(app.staticTexts["Last month"])
```

Comment:
The label is now "Last month".
![Simulator Screenshot - iPhone 15 Pro - 2024-10-29 at 10 26 45](https://github.com/user-attachments/assets/d4c1837a-35df-4ac3-b458-eb19ef83ce25)